indico/modules/events/editing/operations.py

```diff
@@ -153,17 +153,19 @@ def review_editable_revision(revision, editor, action, comment, tags, files=None
     revision.editable.revisions.append(new_revision)
     if action in (EditingReviewAction.update, EditingReviewAction.update_accept):
         _ensure_publishable_files(new_revision)
+    # XXX: all of the submitters of the contribution should be notified here
+    submitter = revision.editable.revisions[0].user
```

Comment:
@ThiefMaster one last thing for this PR (but can be left for later) - is there an easy&clean way to obtain the list of users that has submission privileges for a contribution? i guess getting all persons that have the submission role would work but im not sure if thats the best way

Reply:
I guess you only care about people who explicitly have that privilege, nothing inherited (managers etc)? If yes, look at how `get_manager_list` is implemented. 

In fact, you could simply adapt that method to add `permission` and `explicit` kwargs (which would be forwarded to the `has_management_permission` call) and then use that instead of writing much new coder that's going to be very similar anyway.

Reply:
hmm, I guess what I actually want here is `get_manager_emails` - anyways, i'd say to get this PR merged and add that to the todo list

Reply:
done, ended up using `get_manager_list` since we actually need some more data from the submitters